### ISSUE TITLE
NINA: Implementation of K-Fold Cross-Validation

### ISSUE THREAD

**Issue by ArniGoyal** [GDG-IGDTUW/AI-ML-1#23]
A single train_test_split might give a biased view of performance depending on how the data was shuffled.

**Tasks:** 

- Replace the fixed split with StratifiedKFold (k=5).

- Document if this changes the model accuracy.

**Comment by Tripti-44** [comment]
pls assign me the issue

**Comment by manasvi9745** [comment]
@ArniGoyal please assign this issue to me
